README.md

```diff
@@ -14,7 +14,20 @@
 
 ##Installation Instructions
 
-    composer require silverstripe/wordpressimport 0.3.*
+    composer require thiagovandieten/export-wordpresstosilverstripe *
```

Comment:
True. I need to register this on packagist. I've recently moved the repo so was waiting.